### ISSUE TITLE
 Grpc heartbeat

### ISSUE THREAD

**Issue by pjjwpc** [PlayFab/consuldotnet#124]
Consul already provides a grpc heartbeat checking mechanism, and if consuldotnet can be updated, just add the GRPC attribute to the AgentServiceCheck class

**Comment by zpc870921** [comment]
yeh,I think so!

**Comment by mymorning** [comment]
yes, we also need this function immediately. so please quickly updated. thks
